code/modules/mob/living/carbon/alien/humanoid/queen.dm

```diff
@@ -45,6 +46,17 @@
 	smallsprite.Grant(src)
 	return ..()
 
+/mob/living/carbon/alien/humanoid/royal/queen/get_status_tab_items()
+	. = ..()
+	if(time_to_shuttle)
+		. += ""
+		. += "Blocked Shuttle Timer: [round(timeleft(time_to_shuttle) / 600, 1)] minutes" //weird conversion but works
```

Comment:
You should probably just make this like the normal shuttle timer as it is more consistent with what already exists, plus you don't have to deal with the funny "1 minutes" grammar issue.

Reply:
minutes are fine as this is not a type of info you'd need live updates on, only minutes, plus timeleft does the work just fine